### ISSUE TITLE
Problem with composer

### ISSUE THREAD

**Issue by ljanecek** [codewizz/laravel-reddit-api#6]
When I try to install this awesome vendor via composer:  `$ composer require codewizz/laravel-reddit-api`

The result is:
```

Using version ^0.2.0 for codewizz/laravel-reddit-api
./composer.json has been updated
Running composer update codewizz/laravel-reddit-api
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires codewizz/laravel-reddit-api ^0.2.0 -> satisfiable by codewizz/laravel-reddit-api[0.2.0].
    - codewizz/laravel-reddit-api 0.2.0 requires illuminate/support ^6.0|^7.0 -> found illuminate/support[v6.0.0, ..., 6.x-dev, v7.0.0, ..., 7.x-dev] but these were not loaded, likely because it conflicts with another require.


Installation failed, reverting ./composer.json and ./composer.lock to their original content.
```

And I don' know how to fix it. Thank you for your help.

**Comment by ghaskell** [comment]
The library has been updated for Larvavel 8 in composer but @codewizz has not yet tagged that change.  By default, composer will download the latest tagged released (0.20) which doesn't include the change.  Dev-master does.  

If you require with `composer require codewizz/laravel-reddit-api dev-master` you should be in business until the maintainer tags 0.2.1

**Comment by ljanecek** [comment]
> The library has been updated for Larvavel 8 in composer but @codewizz has not yet tagged that change. By default, composer will download the latest tagged released (0.20) which doesn't include the change. Dev-master does.
> 
> If you require with `composer require codewizz/laravel-reddit-api dev-master` you should be in business until the maintainer tags 0.2.1

Thank you, it works.
